build_scripts_examples/liquibase.flowfile.yaml

```diff
@@ -71,15 +58,32 @@ globalVariables:
   ###
   ### Other settings
   ### LIQUIBASE_COMMAND_CHECKS_RUN_CHECKS_OUTPUT: "issues"
+  ### LIQUIBASE_COMMAND_CHECKS_RUN_CHECKS_SCRIPTS_ENABLED: "true"
   ### LIQUIBASE_SHOW_BANNER: "false"
   ###
-  ### Label filters
+  ### LB_BASE_DIR: REQUIRED
+  ### The directory to write Liquibase output (e.g., logs, reports)
+  ### LB_BASE_DIR: "/liquibase/changelogs"
+  LB_BASE_DIR: "${LB_BASE_DIR:-.}"
```

Comment:
Base dir is .  will this work as a default for docker?  Or is the default run from local cli?

Reply:
Probably not. The default is just for running from CLI.